ui/webui/resources/BUILD.gn

```diff
@@ -164,6 +166,13 @@ copy("static_copy") {
   outputs = [ "$brave_resources_extra_grdps_path/{{source_target_relative}}" ]
 }
 
+copy("leo_static_copy") {
```

Comment:
This needs to be a dep of something, probably `static_grdp`

Reply:
:facepalm: